### ISSUE TITLE
Spam protection

### ISSUE THREAD

**Issue by M-A-Boucher** [StatCan/StatCan.OrchardCore#31]
Currently no spam protection on feedback or namespace features.
Add captcha v3 to protect them and not burden the user.

**Comment by brendangadd** [comment]
I'll throw two other options in here:
* Anti-spam filter
* Limit feedback to users who have signed in

**Comment by jonlamdev** [comment]
* See existing: https://docs.orchardcore.net/en/dev/docs/reference/modules/ReCaptcha/

**Comment by jptissot** [comment]
Yeah, we wanted to implement captcha v3. I guess we could make a contribution to oc

**Comment by jptissot** [comment]
Suggest doing this as a contribution to OC.
